paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -406,36 +406,39 @@ class FCPrimitiveFactory {
   // scaled with its own scales, this data needs to be divided by
   // those scales to normalise them back to what their floating-point range
   // was. Then we multiply them by desired output scale we want on the output.
-  std::tuple<std::vector<float>, float> ComputeOutputShiftScale(
+  std::tuple<float, std::vector<float>, float> ComputeOutputShiftScale(
```

Comment:
Why did you add a new tuple element at the beginning instead of the end?

Reply:
I wanted to follow the existing pattern because similar quantization is done for Conv operator.

Anyway, I delete all int8 changes (because they are not finished) from this PR to make it more concise.